presidio-anonymizer/presidio_anonymizer/anonymizers/validators.py

```diff
@@ -0,0 +1,16 @@
+from presidio_anonymizer.entities import InvalidParamException
+
+
+def validate_parameter(
+    parameter_value, parameter_name: str, parameter_type: type
```

Comment:
@omri374 How do I hint parameter_value?

Reply:
You can either use Union: `x: Union[str, int, float, bool]` if you have a finite list, or just mark this as `noqa` and document what's expected.

Reply:
It's not failing on linting, the list you gave enough is finite enough for json values right?

Reply:
Not sure why it's not failing. You can skip type hints if they are too complex or don't make sense. However you should add `:param`s to docstring

Reply:
Yes the list is finite for scalars (https://www.w3schools.com/js/js_json_datatypes.asp).